### DISCUSSION TITLE
as a vscode user...

### DISCUSSION THREAD

**Discussion by twwwwx** [visualpython/visualpython#144] [Q&A] [answered]
Can I use it in a jupyter environment with vscode? If so how do I use it? If not is it possible to have a vscode extension?Thanks~

**Comment by llbtl** [comment, marked as answer]
Hi @twwwwx, 
We definitely will support VS Code as an extension to Jupyter Notebook extension, however it may take some time because we are currently working on Colab first. Our next steps are as follows : Colab - Jupyter Lab - VS Code. We will let you know as soon as it is ready!

**Reply by twwwwx** [reply]
It's so kind of you!thank you and looking forward to your extensions both on Colab and VScode!

**Comment by MelodyDuplaix** [comment]
Hello, 
Has there been any progress on implementing visual python in VSCode? I'm a big fan of VScode and would be delighted if this extension were available on it.

**Reply by minjk-bl** [reply]
Hello @MelodyDuplaix.
Unfortunately, we are not working on VS Code extension... There're some priorities planned before considering VS Code extension.
I'll let you know if something changes.
